projects/PM_hash_navigation/pm_hasher.h

```diff
@@ -22,7 +25,7 @@ class PM_Hasher
         return count;
     }
 
-    void computeDCT_Hash(cv::Mat &in, unsigned long long &hash)
+    void computeDCT_Hash(const cv::Mat &in, unsigned long long &hash)
```

Comment:
``unsigned long long computeDCT_Hash(const cv::Mat &in)`` would be a more intuitive signature